config/initializers/date_formats.rb

```diff
@@ -1,6 +1,7 @@
 Time::DATE_FORMATS[:govuk_date] = '%-d %B %Y'
 Time::DATE_FORMATS[:govuk_date_short_month] = '%-d %b %Y'
 Date::DATE_FORMATS[:govuk_date] = '%-d %B %Y'
+Date::DATE_FORMATS[:govuk_date_short_month] = '%-d %b %Y'
```

Comment:
I think duplicated from 2 lines up?

Reply:
I had the same thought when I read the first time. But actually, one is for Date and another for Time.